### PR TITLE
Release v0.4.1

### PR DESCRIPTION
**Pull request by michalfita**
Formality.

### CODE REVIEW

**Comment by martinmortsell** [comment]
The usage instructions in  `hal/README.md` should also be updated to the new version number.

**Comment by michalfita** [comment]
@martinmortsell but it is.

**Comment by martinmortsell** [comment]
> @martinmortsell but it is.

Right. Sorry, I had noticed it was still at `0.3.0` earlier but missed that it had been updated here.

**Review by martinmortsell — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,16 @@
 
 ### Changed
 
+### Removed
+
+### Fixed
+
+## [v0.4.1] 2022-10-26
+
+### Added
```
Comment (tmplt) [inline CHANGELOG.md]: Empty sections can be omitted.

**Review by tmplt — CHANGES_REQUESTED** [review]

**Pull request merged**